### ISSUE TITLE
cytolib version loading

### ISSUE THREAD

**Issue by rwbaer** [DillonHammill/CytoExploreR#54]
**Describe the bug**
Don't know if this is directly related to your namespace, but the current master seems to have an issue with cytolib loading (version requirment via flowWorkspace)
**To Reproduce**

```{r}
> # Load required packages
# Make sure you are using the latest code
devtools::install_github("DillonHammill/CytoExploreR")
> library(CytoExploreR)
Loading required package: flowCore
Loading required package: flowWorkspace
Registered S3 method overwritten by 'data.table':
  method           from
  print.data.table     
Error: package or namespace load failed for �flowWorkspace� in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 namespace �cytolib� 2.1.2 is already loaded, but >= 2.1.4 is required
Error: package ‘flowWorkspace’ could not be loaded
```

**Comment by DillonHammill** [comment]
Read the error message, your `cytolib` requires updating to use the latest `flowWorkspace`. Install `devel` version from BioC or GitHub.

**Comment by rwbaer** [comment]
The issue was that I was locked out from doing so.   I had to uninstall multiple packages to get flowWorkspace to update, but it ultimately involved MANUALLY updating cytolib from git which seemed already to have been added from the cytoExploreR update.  

This may be a transient issue involved with the dynamics of development, and I did get it working, but I thought it worth reporting because all I did to cause everything to "crumble" was run devtools::install_github("DillonHammill/CytoExploreR")  

It seemed flowWorkspace wanted a newer cytolib than the one already loaded by CytoExploreR.  You  leave it closed as you see fit.
